xscen/data/file_schema.yml

```diff
@@ -175,16 +252,16 @@ derived-hydro-sims:
   filename: [ variable, xrfreq, hydrology_project, hydrology_institution, hydrology_source, bias_adjust_project, version, mip_era, activity, institution, source, driving_model, experiment, member, domain, processing_level, DATES ]
 derived-hydro-reconstruction:
   with:
-    - option: type
+    - facet: type
       value: reconstruction-hydro
   folders:
     - type
     - hydrology_project
-    - hydrology_institution
     - hydrology_source
+    - (hydrology_member)
     - institution
     - [source, version]
-    - member
+    - (member)
     - domain
     - processing_level
     - xrfreq
```

Comment:
There's also a filename to fix below this line, but I don't have access to it from the PR..